test/cases/generation.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/bash
+set -euxo pipefail
+
+# Provision the software under test.
+/usr/libexec/osbuild-composer-test/provision.sh
+
+# Get OS data
+source /usr/libexec/osbuild-composer-test/set-env-variables.sh
+
+# Colorful timestamped output.
+function greenprint {
+    echo -e "\033[1;32m[$(date -Isecond)] ${1}\033[0m"
+}
+
+# install requirements
+sudo dnf -q -y install make \
+    git \
+    go \
+    ranger \
+    openssl \
+    rpm-build \
+    aws \
+    qemu \
+    setools-console \
+    setroubleshoot \
+    container-selinux \
+    lvm2
```

Comment:
We discussed this and I hoped that you'll eventually trim the list of packages only to those required for this test case to pass as there are definitely some unneeded packages. I'm not insisting that you do this in this PR, a follow up is fine by me. However if you are going to rebase this once more, you may as well clean it up.

I *think* that only `go` should be needed from the list. In addition, there is no reason to make dnf quiet by `-q` as we will at least see the installed package versions.

Reply:
Thanks @thozza Indeed we discussed it and I forgot to trim it. I'll do it in a follow up since I'm able to finally merge this PR. I'll keep you posted.